Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs

```diff
@@ -123,4 +123,124 @@ public void GivenCreateVM_WhenCreatePostIsCalled_ThenModelStateIsTrueAndReturnsR
         _parkingSlotServiceMoq.Verify(x => x.Create(It.IsAny<ParkingSlot>()), Times.Once);
     }
     #endregion
+
+     #region Edit
+        [Fact]
+        public void GivenParkingSlotId_WhenEditGetIsCalled_ThenReturnViewResult()
+        {
+            //Arrange
+            EditVM expectedEditVM = new(_parkingSlot[0]);
+            _parkingSlotServiceMoq.Setup(x => x.GetById(_parkingSlot[0].Id)).Returns(_parkingSlot[0]);
+
+            //Act
+            var result = _parkingSlotController.Edit(_parkingSlot[0].Id);
+
+            //Assert
+            var model = Assert.IsType<ViewResult>(result).Model;
+            Assert.NotNull(result);
+            Assert.Equal(JsonSerializer.Serialize(model), JsonSerializer.Serialize(expectedEditVM));
+            _parkingSlotServiceMoq.Verify(x => x.GetById(_parkingSlot[0].Id), Times.Once);
+        }
+
+        [Fact]
+        public void GivenParkingSlotId_WhenEditGetIsCalled_ThenReturnNotFound()
+        {
+            //Arrange
+            _parkingSlotServiceMoq.Setup(x => x.GetById(_parkingSlot[0].Id));
+
+            //Act
+            var result = _parkingSlotController.Edit(_parkingSlot[0].Id);
+
+            //Assert
+            Assert.NotNull(result);
+            Assert.IsType<NotFoundResult>(result);
+            _parkingSlotServiceMoq.Verify(x => x.GetById(_parkingSlot[0].Id), Times.Once);
+        }
+
+        [Fact]
+        public void GivenEditVMAndParkingSlotId_WhenEditPostIsCalled_ThenReturnNotFoundIfIdAndSlotIdDoesNotMatch()
```

Comment:
NamingConvention error 
```suggestion
        public void GivenEditVMAndParkingSlotId_WhenEditPostIsCalled_ThenIdAndSlotIdDoNotMatchAndReturnsNotFound()
```